src/details/ArborX_DetailsKokkosExtViewHelpers.hpp

```diff
@@ -0,0 +1,86 @@
+/****************************************************************************
+ * Copyright (c) 2017-2022 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef ARBORX_DETAILS_KOKKOS_EXT_VIEW_HELPERS_HPP
+#define ARBORX_DETAILS_KOKKOS_EXT_VIEW_HELPERS_HPP
+
+#include <Kokkos_Core.hpp>
+
+namespace KokkosExt
+{
+
+// FIXME it is not legal to use KOKKOS_IMPL_CTOR_DEFAULT_ARG
+template <class ExecutionSpace, class View>
+void reallocWithoutInitializing(ExecutionSpace const &space, View &v,
```

Comment:
I would prefer that we forward to the `Kokkos` versions if the `Kokkos` version used allows.

Reply:
Yes that what I want to do eventually.  Wasn't planning on doing it in this PR.